grpc/server/dronecore_server.cpp

```diff
@@ -21,16 +28,34 @@ using grpc::ServerReaderWriter;
 using grpc::ServerWriter;
 using grpc::Status;
 using dronecorerpc::Empty;
+using dronecorerpc::ActionEmpty;
+using dronecorerpc::MissionEmpty;
+using dronecorerpc::TelemetryEmpty;
 using dronecorerpc::DroneCoreRPC;
+using dronecorerpc::ActionRPC;
+using dronecorerpc::MissionRPC;
+using dronecorerpc::TelemetryRPC;
 
 using namespace dronecore;
 using namespace std::placeholders;
 
+static DroneCore dc;
+
+template<typename T> ::grpc::Service *createInstances() {return new T;}
+
+typedef std::map<std::string, ::grpc::Service*(*)()> map_type;
+
 class DroneCoreRPCImpl final : public DroneCoreRPC::Service
 {
 
 public:
-    Status Arm(ServerContext *context, const Empty *request,
+
+};
+
+class ActionRPCImpl final : public ActionRPC::Service
```

Comment:
I think these classes should be in separate files and then inside the respective plugin folder.